app/controllers/decidim/action_delegator/admin/exports_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ActionDelegator
+    module Admin
+      class ExportsController < ActionDelegator::Admin::ApplicationController
```

Comment:
I want to namespace or rename the class so it includes `Consultation` in it otherwise you might think it's a sort of generic controller that deals with all kinds of exports.